extensions/skyportal/static/js/components/ZTFAlertsByCoords.jsx

```diff
@@ -0,0 +1,372 @@
+import CircularProgress from "@material-ui/core/CircularProgress";
+import MUIDataTable from "mui-datatables";
+
+import ThumbnailList from "./ThumbnailList";
+import { ra_to_hours, dec_to_dms } from "../units";
+import * as Actions from "../ducks/alertsByCoords";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: "100%",
+  },
+  container: {
+    maxHeight: 440,
+  },
+  whitish: {
+    color: "#f0f0f0",
+  },
+  itemPaddingBottom: {
+    paddingBottom: "0.5rem",
+  },
+  saveAlertButton: {
+    margin: "0.5rem 0",
+  },
+  image: {
+    padding: theme.spacing(1),
+    textAlign: "center",
+    color: theme.palette.text.secondary,
+  },
+  paper: {
+    padding: theme.spacing(2),
+    textAlign: "center",
+    color: theme.palette.text.secondary,
+  },
+  heading: {
+    fontSize: "1.0625rem",
+    fontWeight: 500,
+  },
+  header: {
+    paddingBottom: "0.625rem",
+    color: theme.palette.text.primary,
+  },
+
+  accordionHeading: {
+    fontSize: "1.25rem",
+    fontWeight: theme.typography.fontWeightRegular,
+  },
+  accordionDetails: {
+    width: "100%",
+  },
+  source: {
+    padding: "1rem",
+    display: "flex",
+    flexDirection: "row",
+  },
+  column: {
+    display: "flex",
+    flexFlow: "column nowrap",
+    verticalAlign: "top",
+    flex: "0 2 100%",
+    minWidth: 0,
+  },
+  columnItem: {
+    margin: "0.5rem 0",
+  },
+  name: {
+    fontSize: "200%",
+    fontWeight: "900",
+    color: "darkgray",
+    paddingBottom: "0.25em",
+    display: "inline-block",
+  },
+  alignRight: {
+    display: "inline-block",
+    verticalAlign: "super",
+  },
+  sourceInfo: {
+    display: "flex",
+    flexFlow: "row wrap",
+    alignItems: "center",
+  },
+  position: {
+    fontWeight: "bold",
+    fontSize: "110%",
+  },
+}));
+
+function isString(x) {
+  return Object.prototype.toString.call(x) === "[object String]";
+}
+
+const getMuiTheme = (theme) =>
+  createMuiTheme({
+    overrides: {
+      MUIDataTableBodyCell: {
+        root: {
+          padding: `${theme.spacing(0.25)}px 0px ${theme.spacing(
+            0.25
+          )}px ${theme.spacing(1)}px`,
+        },
+      },
+    },
+  });
+
+const ZTFAlertsByCoords = ({ route }) => {
+  const { ra, dec, radius } = route;
+  const dispatch = useDispatch();
+  const history = useHistory();
+  const [isFetched, setIsFetched] = useState(false);
+
+  const userAccessibleGroups = useSelector(
+    (state) => state.groups.userAccessible
+  );
+
+  const userAccessibleGroupIds = useSelector((state) =>
+    state.groups.userAccessible?.map((a) => a.id)
+  );
+
+  const theme = useTheme();
+  const darkTheme = theme.palette.type === "dark";
+
+  const alerts = useSelector((state) => state.alertsByCoords);
+
+  const makeRow = (alert) => {
+    return {
+      objectId: alert?.objectId,
+      candid: alert?.candid,
+      jd: alert?.candidate.jd,
+      ra: alert?.candidate.ra,
+      dec: alert?.candidate.dec,
+      drb: alert?.candidate.drb,
```

Comment:
Let's also render the acai scores:

```suggestion
      drb: alert?.candidate.drb,
      acai_h: alert?.classifications?.acai_h,
      acai_n: alert?.classifications?.acai_n,
      acai_o: alert?.classifications?.acai_o,
      acai_v: alert?.classifications?.acai_v,
      acai_b: alert?.classifications?.acai_b,
```